sdk/personalizer/azure-ai-personalizer/mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy]
```

Comment:
You'll need to enable mypy CI checks by adding your library to this file: https://github.com/Azure/azure-sdk-for-python/blob/main/eng/tox/mypy_hard_failure_packages.py